### ISSUE TITLE
Fix NPM errors in Travis CI build

### ISSUE THREAD

**Issue by webinista** [department-of-veterans-affairs/vets-website#2498]
Possible solution is to add `npm cache clean` to Rakefile.

**Comment by carlmjohnson** [comment]
I installed vets-website, but ran into a few issues with node (I think webpack needs to be globally installed?). In attempt to document the issue, I cloned the repo into a fresh folder and ran rake install again. This time, npm has just been hanging during install. I imagine that Travis is probably having the same problems. 

It seems like a lot of things are being installed by npm. Can we audit them and remove the cruft? I think reducing the number of dependencies would make isolating any remaining problems simpler.
